frontend/src/components/TemplatesIndex/TemplatesIndex.tsx

```diff
@@ -55,15 +70,64 @@ export const TemplatesIndex: FC<TemplateIndexProps> = () => {
             }
             return new Date(date).toLocaleDateString()
 
+        },
+        'lastUpdated': (d) => {
+            const date = Date.parse(d)
+            if (isNaN(date)) {
+                return new Date().toLocaleDateString()
+            }
+            return new Date(date).toLocaleDateString()
+
         }
     }
 
     const templateColumns = [
-        'name', 'updatedAt', 'createdBy', 'labName', 'status'
+        'name', 'lastUpdated', 'createdBy', 'lab', 'status', 'updatedAt', 'labName'
     ]
 
-
-    if (!templateQuery.data  || templateQuery.data.length === 0) {
+    useEffect(() => {
```

Comment:
We could very safely not have this at all. Template creation works and it was really only there for demoing the UI of the template page.